index.yaml

```diff
@@ -100,3 +100,9 @@ sources:
     license: GPLv3
     url: https://raw.githubusercontent.com/travisbgreen/hunting-rules/master/hunting.rules
     min-version: 4.1.0
+
+versions:
+
+  suricata:
+    recommended: 4.1.4
+    "4.1": 4.1.4
```

Comment:
I think we're missing the block for `suricata-update` versions here?

Reply:
@shivan1b Could you please help me with the release of suricata-update?
Is the recommended version: `1.1.0dev0`? Has `1.0` reached EOL?

Reply:
@vagishagupta23 the recommended version should be the latest `1.0.5`.